### PR TITLE
upload test scripts for finding streamorder

### PR DESCRIPTION
**Pull request by ellenbechtel**
These files are pretty messy!  But the one to look at is gage_streamorder > [gage-streamorder.R](https://github.com/ellenbechtel/viz-scratch/blob/master/gage_streamorder/gage-streamorder.R)

### CODE REVIEW

File: gage_streamorder/gage-streamorder.R [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+readNWISsite(gage_list[1]) # for a single site
+gage_da <- readNWISsite(gage_list) # for a all sites
+
+# Pick a random single gage!
+rand <- 10000
+
+# grab coords of a single gage and convert to comid for NHDPLus
+start <- gage_da %>% slice(rand)
+start_point <- st_sfc(st_point(c(start$dec_long_va,start$dec_lat_va)), crs = 4269)
+comid <- discover_nhdplus_id(start_point)
```
Comment (collnell) [inline gage_streamorder/gage-streamorder.R:24]: Try turning these two lines into a function that takes the gage number or whatever gage input data we have, and returns the comid.

File: gage_streamorder/gage-streamorder.R [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+# get the next closest comid within 10 km
+nldi_feature <- list(featureSource = "comid", featureID = start_comid)
+flowline_nldi <- navigate_nldi(nldi_feature, 
+                               mode = "upstreamTributaries", 
+                               distance_km = 100)
+next_comid <- flowline_nldi$UT_flowlines %>% slice(1)
+
+# assign to a variable
```
Comment (collnell) [inline gage_streamorder/gage-streamorder.R:33]: I think these lines will be unnecessary per our conversation about stream order. Alternatively, another function that takes the comid and runs the lines below will download nhd data for each comid, including a variable called `stream_orde`

**Comment by collnell** [comment]
First, let's remove/gitignore all files except the script `gage_streamoder.R` and put that one in a folder names `src`. The `.rds` file with the data can go in another folder `in` but let's store it locally and not commit to the repo. 

Then, we're going to need to set up a pipeline here to help manage the data pulls for a large number of gages/comids. To do that, you can adapt the `gage_streamorder.R` script into functions. The first one should be one that accepts a gage site_no and returns the comid. Once that is ready, I can add the pipeline to run these functions for all the gage sites.

**Comment by ellenbechtel** [comment]
can do, especially now that I'm getting familiar with pipelines again with this training! thankssssss

**Comment by ceenell** [comment]
Closing this because it's died.

**Pull request closed without merging**